### ISSUE TITLE
Broken links - Missing Files

### ISSUE THREAD

**Issue by kfriesth** [erlerobot/HRIM#26]
Broken links and missing files from docs

ie   https://acutronicrobotics.com/docs/technology/hrim/model/implementations/actuator/rotary%20servomotor

has a link like

https://github.com/erlerobot/HRIM/blob/master/actuator/RotaryServo/hrim_actuator_rotaryservo_msgs/msg/GoalRotaryServo.msg  

IE 404 error

Any links to the former blob directory path and beyond from your website doc links now breaks all your links from your website to the support file links and you have lots links that have 404 page errors

you might want to fix that as it makes it hard to follow along

**Comment by ibaiape** [comment]
Thanks for communicating this!

With our jump to XML models the whole repository structure has changed. One of these changes was removing an already generated implementation of HRIM.

The whole *Implementations* subsection on our docs was hidden to reflect this change. I'm guessing you got to that page through an external link, am I right? It shouldn't be accessible anymore.

Our artifact definitions currently sit at our *Models* subsection, in XML form. Adding each .msg/.srv definition on the webpage is something we are considering, any input on this?

Anyhow, if you are getting started with HRIM may I suggest our [tooling](https://acutronicrobotics.com/docs/technology/hrim/model/tooling) section (after getting through our [general structure](https://acutronicrobotics.com/docs/technology/hrim/introduction/general%20structure) and [component model](https://acutronicrobotics.com/docs/technology/hrim/introduction/hrim%20component%20model) pages at least).

Thanks again for informing us on this!
